is-state/is-state.typ

```diff
@@ -0,0 +1,227 @@
+#import "@preview/glossarium:0.4.1": gls, glspl
+
+= Ist-Zustand
+
+Das originale Projekt, auf dem unsere Arbeit basiert, kann als öffentliches GitHub Repository hier gefunden werden: #link("https://github.com/lost-university/web").
+
+Wir haben uns dazu entschieden, einen Fork von diesem Projekt zu erstellen, auf welchem wir unsere Anpassungen durchführen. Dies damit allfällige Änderungen am Original, welche nach dem Start unserer Arbeit anfallen, keinen direkten Einfluss auf unsere Arbeit haben.
+
+Ein eigener Fork hat auch den Vorteil gegenüber einem Branch, dass das Hosting einfacher zu bewerkstelligen ist.
```

Comment:
Hier könnten wir noch anfügen "wie erklärt in Abschnitt xx" und dann auf das Kapitel verweisen, in welchem das Hosting erklärt wird (Architektur?)